.stylelintignore

```diff
@@ -5,3 +5,22 @@
 # Unignore css and scss files too.
 !**/*.css
 !**/*.scss
+
+# Ignore minifed CSS
+**/*.min.css
+
+# Ignore CSS copied in from elsewhere
+projects/js-packages/components/components/boost-score-graph/style-uplot.scss
```

Comment:
> And if we import it from the js file, it breaks jest.

That should be fixed. I'm pretty sure we do it elsewhere without breaking jest.

Reply:
I'll look at this separately.

Reply:
For the record, it needs this added to the jest config to be able to import from the package without breaking jest
```
    transformIgnorePatterns: [
        '/node_modules/(?!\.pnpm|uplot/dist/uPlot.min.css)',
    ],
```
More details at p1743610653235219-slack-C05Q5HSS013